src/components/StyledProgress/StyledProgress.tsx

```diff
@@ -59,53 +69,54 @@ const StyledProgress = ({
   secondary = 0,
   receivedBelow = '',
   committedBelow = '',
+  barHeight = 46,
 }: Props): ReactElement => {
   const locale = useLocale();
   const { t } = useTranslation();
-  const { classes } = useStyles();
-
   return (
-    <Box className={classes.box}>
+    <ProcessBoxContainer barHeight={barHeight}>
       {loading ? (
-        <Skeleton
+        <StyledSkeleton
           data-testid="styledProgressLoading"
-          className={classes.skeleton}
+          barHeight={barHeight + 'px'}
```

Comment:
Numbers in styles are treated as pixels, so we should be able to just use `barHeight` instead of `barHeight + 'px'` here and the two other places below.